test/tests/val-2b/comparison_val-2b.py

```diff
@@ -73,5 +73,5 @@ def numerical_solution_on_experiment_input(experiment_input, tmap_input, tmap_ou
 RMSPE = RMSE*100/np.mean(expt_flux)
 ax.text(870,3e16, 'RMSPE = %.2f '%RMSPE+'%',fontweight='bold')
 ax.minorticks_on()
-plt.savefig('val-2b_comparison.png', bbox_inches='tight')
+plt.savefig('val-2b_comparison.png', bbox_inches='tight', dpi=300)
```

Comment:
So high def!

Reply:
Too much for your taste? I get tired of pixelated figures. It's also convenient to copy figures to presentations without worrying about the resolution.

Reply:
No, I am OK with this 😄 Just goofing around. Since we generate the figures on the fly now, there is no concern about storage!